grammar.js

```diff
@@ -91,83 +91,112 @@ module.exports = grammar({
+    decimal_integer_literal: _ => token(seq(
       DIGITS,
-      optional(choice('l', 'L'))
+      optional(choice('l', 'L')),
     )),
 
-    hex_integer_literal: $ => token(seq(
+    hex_integer_literal: _ => token(seq(
       choice('0x', '0X'),
       HEX_DIGITS,
-      optional(choice('l', 'L'))
+      optional(choice('l', 'L')),
     )),
 
-    octal_integer_literal: $ => token(seq(
+    octal_integer_literal: _ => token(seq(
       choice('0o', '0O'),
       sep1(/[0-7]+/, '_'),
-      optional(choice('l', 'L'))
+      optional(choice('l', 'L')),
     )),
 
-    binary_integer_literal: $ => token(seq(
+    binary_integer_literal: _ => token(seq(
       choice('0b', '0B'),
       sep1(/[01]+/, '_'),
-      optional(choice('l', 'L'))
+      optional(choice('l', 'L')),
     )),
 
-    decimal_floating_point_literal: $ => token(choice(
+    decimal_floating_point_literal: _ => token(choice(
       seq(DIGITS, '.', optional(DIGITS), optional(seq((/[eE]/), optional(choice('-', '+')), DIGITS)), optional(/[fFdD]/)),
       seq('.', DIGITS, optional(seq((/[eE]/), optional(choice('-', '+')), DIGITS)), optional(/[fFdD]/)),
       seq(DIGITS, /[eEpP]/, optional(choice('-', '+')), DIGITS, optional(/[fFdD]/)),
-      seq(DIGITS, optional(seq((/[eE]/), optional(choice('-', '+')), DIGITS)), (/[fFdD]/))
+      seq(DIGITS, optional(seq((/[eE]/), optional(choice('-', '+')), DIGITS)), (/[fFdD]/)),
     )),
 
-    hex_floating_point_literal: $ => token(seq(
+    hex_floating_point_literal: _ => token(seq(
       choice('0x', '0X'),
       choice(
         seq(HEX_DIGITS, optional('.')),
-        seq(optional(HEX_DIGITS), '.', HEX_DIGITS)
+        seq(optional(HEX_DIGITS), '.', HEX_DIGITS),
       ),
       optional(seq(
         /[eEpP]/,
         optional(choice('-', '+')),
         DIGITS,
-        optional(/[fFdD]/)
-      ))
+        optional(/[fFdD]/),
+      )),
     )),
 
-    true: $ => 'true',
+    true: _ => 'true',
 
-    false: $ => 'false',
+    false: _ => 'false',
 
-    character_literal: $ => token(seq(
-      "'",
+    character_literal: _ => token(seq(
+      '\'',
       repeat1(choice(
         /[^\\'\n]/,
         /\\./,
-        /\\\n/
+        /\\\n/,
       )),
-      "'"
-    )),
-
-    string_literal: $ => token(choice(
-      seq('"', repeat(choice(/[^\\"\n]/, /\\(.|\n)/)), '"'),
-      // TODO: support multiline string literals by debugging the following:
-      // seq('"', repeat(choice(/[^\\"\n]/, /\\(.|\n)/)), '"', '+', /\n/, '"', repeat(choice(/[^\\"\n]/, /\\(.|\n)/)))
+      '\'',
     )),
 
-    text_block: $ => token(
-      seq(
-        /"""\s*\n/,
-        optional(/([^"]|("[^"])|(""[^"]))+/),
-        '"""'
-      )
+    string_literal: $ => choice($._string_literal, $._multiline_string_literal),
+    _string_literal: $ => seq(
+      '"',
+      repeat(choice(
+        $.string_fragment,
+        $.escape_sequence,
+      )),
+      '"',
     ),
+    _multiline_string_literal: $ => seq(
```

Comment:
I just want to see those modifications in the PR.

Reply:
I agree, my editor has an auto format on save which added the semicolons/commas and such, it is also a bit irking having the inconsistency of some comma sep items end in commas and some not. After this I'll submit a general PR adding the linting and other fixes across the board if whatever was in there before is ok w/ you.